frontend/src/components/Header/Header.js

```diff
@@ -1,38 +1,34 @@
 import React, { Component } from 'react';
-import "./Header.css"
-
+import './Header.css';
 
 class Header extends Component {
-    constructor(props) {
-        super(props)
-        this.state = {
-            state: " "
-        }
-    }
-
-    render() {
-        return (
-            <div className="container-fluid">
-              <header className="navbar navbar-white bg-white fixed-top">
-                <div className="row col-sm-2 col-md-2">
-                    <a className="navbar-brand" href="/">
-                        <img src={require('../../assets/RLC_logo.png')} alt="RLC Logo" className="rlcLogo"/>
-                    </a>
-                </div>
-                <div className="row offset-1 col-sm-3 col-md-3">
-                    {/* px-4 add padding on left and right */}
-                    <a className="btn navbar-btn px-4 topButton browseButton" href="/">Browse Recipes</a>
-                </div>
-                <div className="row col-sm-2 col-md-2">
-                    <a className="btn navbar-btn px-4 topButton loginButton" href="/">Log In or Sign Up</a>
-                </div>
-                <div className="row col-sm-2 col-md-2">
-                    <a className="btn navbar-btn px-4 topButton donateButton" href="/">Donate</a>
-                </div>
-              </header>
-            </div>
-        )
-    }
-}
+  
+  render() {
+    return (
+      <nav className="navbar navbar-white bg-white fixed-top">
+        <a className="navbar-brand" href="/">
+          <img
+            src={require('../../assets/RLC_logo.png')}
+            alt="RLC Logo"
+            className="rlcLogo"
+          />
+        </a>
+        <div class="d-flex justify-content-lg-end">
+          <button className="btn navbar-btn btn-lg browseButton" href="/">
+            Browse Recipes
+          </button>
+          <button className="btn navbar-btn  btn-lg loginButton ml-3" href="/">
+            Log In or Sign Up
+          </button>
+          <button
+            className="btn navbar-btn ml-4 px-4 btn-lg donateButton"
+            href="/"
+          >
+            Donate
+          </button>
+        </div>
+      </nav>
+    );
+  }
```

Comment:
I think there's a missing closing brace here. The render function is closed, but the class is still open, which is causing CI to fail.

Reply:
should be good now